fbpic/lpa_utils/laser/laser_profiles.py

```diff
@@ -920,8 +923,10 @@ def E_field( self, x, y, z, t ):
         See the docstring of LaserProfile.E_field
         """
         # Perform interpolation from envelope data
-        env = self.interp_function(x, y, t)
+        env = self.interp_function(x, y, (t-self.t_start) )
         # Add laser oscillations
-        E = (env * np.exp(-1.j*self.omega*t)).real
+        E = (env * np.exp(
+            -1.j*self.omega * (t - self.t_start + self.t_min_lasy)
```

Comment:
I would do the same thing as in WarpX, i.e. ignoring the `tmin` from the file. We can discuss this more offline.
```suggestion
            -1.j*self.omega * (t - self.t_start)
```